### PR TITLE
[MOS-447] Clear all call log option

### PR DESCRIPTION
**Pull request by marziel**
**Description**

The main change: added a new dialog enabling the user to clear the whole call log at once. The dialog is opened via pressing the right joystick button in the call log window. The option is marked with a waste bin and an arrow icons.

Auxiliary changes:
* fix of a typo in a build script,
* changed a misleading log message (see below).
Prior to the log message change, when the call log database was empty, an error log about a null pointer returned from the DB was printed upon entering the call log window. This was flawed for two reasons:
      1. there was no null pointer - the DB was just empty,
      2. when taking into account the first remark: then, the situation isn't an error.
The wrong info has been replaced with the right one and the log level decreased to warning.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [x] Has unit tests if possible.
- [x] Has documentation updated
- [x] Don't merge this PR until the waste bin icon is added to the asset repo.

### CODE REVIEW

**Review by Maciej-Mudita — APPROVED** [review]

File: module-gui/gui/widgets/header/DeleteAction.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        : HBox(parent, 0, 0, style::header::navigation_indicator::w, style::header::navigation_indicator::h)
+    {
+        setAlignment(Alignment(Alignment::Horizontal::Right, Alignment::Vertical::Center));
+        setEdges(RectangleEdge::None);
+        setMargins(Margins(style::header::navigation_indicator::right_left_margin,
+                           style::header::navigation_indicator::top_margin,
+                           0,
+                           0));
+
+        auto wasteBin        = new gui::Image("calllog_delete_24px_W_G");
```
Comment (SP2FET) [inline module-gui/gui/widgets/header/DeleteAction.cpp:23]: this file is in generic `module-gui`, so maybe it's worth renaming the asset to be also generic?

**Review by SP2FET — APPROVED** [review]

File: module-gui/gui/widgets/header/DeleteAction.cpp (hunk 1 above)
Comment (marziel) [reply module-gui/gui/widgets/header/DeleteAction.cpp:23]: You mean ust erasing the `calllog_` prefix?

Comment (SP2FET) [reply module-gui/gui/widgets/header/DeleteAction.cpp:23]: exactly ;)

Comment (marziel) [reply module-gui/gui/widgets/header/DeleteAction.cpp:23]: Sure

Comment (marziel) [reply module-gui/gui/widgets/header/DeleteAction.cpp]: Done

Comment (marziel) [reply module-gui/gui/widgets/header/DeleteAction.cpp]: Done :)

File: image/assets/lang/English.json [hunk 2]
```diff
@@ -166,6 +166,8 @@
   "app_calllog_date": "Date",
   "app_calllog_options_delete_call": "Delete call",
   "app_calllog_delete_call_confirmation": "Delete this call from the list?",
+  "app_calllog_delete_all_calls": "Delete all calls",
+  "app_calllog_delete_all_calls_confirmation": "Delete all calls from the list?",
```
[the remark this answers is not in the document]
Comment (SP2FET) [reply image/assets/lang/English.json:170]: sure, but make sure it will be merged before release ;)

**Pull request merged**